src/features/view-subdomains/SubdomainTable/SubdomainTable.tsx

```diff
@@ -43,6 +47,18 @@ export const SubdomainTable: FC<SubdomainTableProps> = ({ zna }) => {
 		setSearchQuery(undefined);
 	}, [zna]);
 
+	useEffect(() => {
```

Comment:
Nice. Will have to add similar to staking tables at some point to switch views rather current implementation.